### ISSUE TITLE
bun is creating files for .env values rather than just reading them

### ISSUE THREAD

**Issue by jdion84** [oven-sh/bun#4727]
### What version of Bun is running?

1.0.0+822a00c4d508b54f650933a73ca5f4a3af9a7983

### What platform is your computer?

Darwin 22.6.0 arm64 arm

### What steps can reproduce the bug?

Create a `.env` file:

```env
APP_PORT=8080
```

Read the value via `server.ts`:

```ts
import express from "express";

const app = express();

const port = process.env.APP_PORT;

app.listen(port, () => {
  console.log(`Listening on http://localhost:${port}`);
});

```

Run `server.ts`:

```console
bun run server.ts
```

Bun creates a file called `8080` in my root folder rather than just reading it

### What is the expected behavior?

read the env value, dont make an unopenable file for it

**Comment by hyuzea** [comment]
I guess this related to #4582 and already [fixed](/oven-sh/bun/commit/ffe4f561a3af53b9f5a41c182de55d7199b5d692) recently. try upgrade to the latest with `bun upgrade --canary`

**Comment by luisjrez** [comment]
@yus-ham i was trying to install the canary build but when i do `bun --version` show's me that i'm using the 1.0.

How can i use the canary version locally?? 🤔

**Comment by hyuzea** [comment]
@luisjrez, you should do `bun --revision` to get version including commit hash info

**Comment by colinhacks** [comment]
@jdion84 Can you check if this is still an issue on the latest version?

**Comment by Electroid** [comment]
This was fixed by #4618
